### PR TITLE
Changes for consistency on NLPModels

### PR DESCRIPTION
**Pull request by abelsiqueira**
Change 1e20 to Inf and -1e20 to -Inf.
Change hess related functions to use keyword y.

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -44,5 +46,3 @@ for p in problems
   end
   cutest_finalize(nlp)
 end
-
-include("test_mpb.jl")
```
Comment (dpo) [inline test/runtests.jl]: I liked it here because it's a nice-looking conclusion to the tests ;-).

File: test/test_mpb.jl [hunk 2]
```diff
@@ -10,3 +10,4 @@ model = NLPtoMPB(nlp, IpoptSolver())
 @assert isa(model, Ipopt.IpoptMathProgModel)
 MathProgBase.optimize!(model)
 @assert MathProgBase.getobjval(model) ≈ 0.0
+cutest_finalize(nlp)
```
Comment (dpo) [inline test/test_mpb.jl]: Good catch!

**Comment by dpo** [comment]
Looks good!

**Comment by dpo** [comment]
I noticed this earlier: https://travis-ci.org/JuliaOptimizers/CUTEst.jl/jobs/141084034#L3870
Only on Linux and seems intermittent.

**Comment by abelsiqueira** [comment]
I think it may be related to v0.5. The error is in a matrix product and the dimensions should match. This may be related to global scope, or reloading the definitions.
Maybe we should use allow_failure for nightly?

**Comment by dpo** [comment]
Is it a bug we can report? If not, I guess we can use `allow_failure`.

**Comment by abelsiqueira** [comment]
Would have to investigate further and try to create a MFE. Don't have much time for that now.

**Comment by dpo** [comment]
It's intermittent. Now the test passed.

File: src/CUTEst.jl [hunk 3]
```diff
@@ -126,6 +126,11 @@ function CUTEstModel(name :: ASCIIString; decode :: Bool=true, verbose = false)
   end
   @cutest_error
 
+  for lim in Any[bl, bu, cl, cu]
+    I = find(abs(lim) .>= 1e20)
+    lim[I] = Inf * lim[I]
+  end
+
```
Comment (dpo) [inline src/CUTEst.jl]: I'm not so sure how CUTEst subroutines will handle those. What will Julia pass Fortran in place of those `Inf`s?

**Comment by abelsiqueira** [comment]
> I noticed this earlier: https://travis-ci.org/JuliaOptimizers/CUTEst.jl/jobs/141084034#L3870
> Only on Linux and seems intermittent.

It happened on the OSX too:
https://travis-ci.org/abelsiqueira/CUTEst.jl/jobs/141866002#L868

File: src/CUTEst.jl (hunk 3 above)
Comment (abelsiqueira) [inline src/CUTEst.jl]: From my tests, Fortran gets an `Infinity`. It's a little confusing to understand if Infinity is always present on Fortran, but it looks like it is on gfortran.
To compare
````Fortran
real*4 x, y
x = 1.0
y = 0.0
write (*,*) x/y
````
returns Infinity.

**Comment by abelsiqueira** [comment]
Rebased

**Pull request merged**